2.0/flask_app/services/llm_service.py

```
@@ -0,0 +1,64 @@
+            f"Filer: {res['company_name']} ({res['filing_date']})\n"
+            f"Filing type: {res['filing_type']}\n"
+            f"Section name: {res['section_name']}\n"
+            f"Content: {res['text'][:1000]}...\n" # TODO summarize instead of truncate
+            f"Relevance: {res['score']:.2f}"
+            for res in top_results
+        )
+        logging.info(f'Built query context string: {context}')
+        
+        # 3. Get LLM response
+        answer = self.query_ollama(user_query, context)
+        logging.info(f'Ollama answered, returning to user.')
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)